index.html

```diff
@@ -32,8 +33,13 @@
 
         <nav class="nav2">
```

Comment:
Kinldy make the necessary changes so that your design resembles as much as possible the given design 🙌 As you can see, the mobile navbar has a transparent background and the visual transition between the button and the transition for the next section is seamless 

Cindy's Design
![image](https://user-images.githubusercontent.com/15807343/163242161-ea049dd1-1b26-4dbf-b804-7cb6e98c0c5f.png)

Your Design
![image](https://user-images.githubusercontent.com/15807343/163242509-5f30e2f9-b7a6-444e-8155-e9b6ecf00630.png)

Reply:
On desktop size, be sure that your logo is aligned to the left of the screen 🙌

Your design 👀
![image](https://user-images.githubusercontent.com/15807343/163243023-036dc61d-8e45-4733-9ec3-b130acaece71.png)

Cindy's design 
![image](https://user-images.githubusercontent.com/15807343/163243094-ce61bda7-3d5a-4ea8-acc1-dc86dde66383.png)

Reply:
Done